### ISSUE TITLE
[Feature request] supporting basic text coloring

### ISSUE THREAD

**Issue by abdurrahman-adel** [JasonStein/Notepads#940]
### i really love this app, and it's awesome

i hope developers of this app support **text coloring** for main or frequently opened files like _yaml, json, js, dart. cpp, java_

**Comment by Plutonium-239** [comment]
Yes, I think basic JSON support like collapsing/expanding trees/lists could be looked into. For syntax highlighting: I don't this is meant to be a code editor, so if you want a really good lightweight code editor, use Sublime Text.
Since @JasonStein has mentioned they want to keep it light, these could be served as add-ons/plugins for notepads.
